Standard.AI.Data.EntityIntelligence/Models/Data/TableColumnMetadata.cs

```diff
@@ -0,0 +1,37 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+
+namespace Standard.AI.Data.EntityIntelligence.Models.Data
+{
+    public class TableColumnMetadata
+    {
+        public string TableSchema { get; }
+
+        public string TableName { get; }
+
+        public string Name { get; }
+
+        public string Type { get; }
+
+        public int? MaxLength { get; }
+
+        public bool IsNullable { get; }
+
+        public bool IsPrimaryKey { get; }
+
+        public short? Ordinal { get; }
+
+        public bool IsForeignKey { get; }
```

Comment:
I think this model has way more attributes than needed for EI.NET.

Reply:
My idea was to have all those attributes that would be necessary in the real world scenario. 
For an MVP we can map only needed ones in the Foundation Service but having any real world scenario in my mind I don't think it would make sense using less than this. 
Model covers mostly what every database has nowadays.

Reply:
Let's stay only with the needed ones for now -

Reply:
What do you see as needed ones?